src/main/java/javaserver/RequestHandler.java

```diff
@@ -24,34 +17,13 @@ public Response handleRequest() {
         String method = request.getMethod();
 
         if (method.equals("GET")) {
-            getRequest(request);
-            return response;
-        } else if (method.equals("POST")) {
-            postRequest(request);
-            return response;
-        } else {
-            return response;
-        }
-    }
-
-    public void postRequest(Request request) {
-        try {
-            Path path = Paths.get("test");
-            byte[] bodyData = request.getBody().getBytes();
-            Files.write(path, bodyData);
-        } catch (IOException e) {
-            System.out.println(e.toString());
+            response = new Get().handleRequest(request);
```

Comment:
It's a bit confusing to me what "Get" is supposed to do - off the bat. I think a more descriptive name like "GetRequestHandler" may help understand what this is for. Same with the other request handlers